neptune/new/internal/operation_processors/async_operation_processor.py

```diff
@@ -89,24 +90,59 @@ def flush(self):
     def start(self):
         self._consumer.start()
 
+    def _wait_for_queue_empty(self, initial_queue_size: int, seconds: Optional[float]):
+        waiting_start = monotonic()
+        max_wait_time = self.STOP_QUEUE_MAX_TIME_NO_CONNECTION_SECONDS if seconds is None else seconds
+        if initial_queue_size > 0:
+            if self._consumer.last_backoff_time > 0:
+                click.echo(f"We have been experiencing connection interruptions during your run. "
+                           f"Client will now try to resume connection and sync data for the next "
+                           f"{max_wait_time} seconds. "
+                           f"You can also synchronize manually using neptune sync command.",
```

Comment:
How about "You can also kill this process and synchronize your data manually later using `neptune sync` command."?